README.md

```diff
@@ -178,12 +178,38 @@ Notes:
 * The AutoScaling groups it runs against can be in any region, since all regions
   are processed at runtime.
 
+#### Installation options ####
+
+The CloudFormation stack takes a number of parameters, which allow you to
```

Comment:
I'm not a native English speaker, so I may be wrong too, but now that you mentioned this I think you are right, I'll fix this.